apple/RNCWebViewManager.m

```diff
@@ -284,17 +297,19 @@ - (BOOL)          webView:(RNCWebView *)webView
 
 RCT_EXPORT_METHOD(releaseWebView:(nonnull NSString *)webViewKey)
 {
-  NSMutableDictionary *sharedWKWebViewDictionary = [[RNCWKWebViewMapManager sharedManager] sharedWKWebViewDictionary];
-  NSMutableDictionary *sharedRNCWebViewDictionary= [[RNCWebViewMapManager sharedManager] sharedRNCWebViewDictionary];
-  
-  sharedWKWebViewDictionary[webViewKey] = nil;
-  
-  RNCWebView *rncWebView = sharedRNCWebViewDictionary[webViewKey];
+  dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
I updated this to always run on the main thread because I was running into this error.

![Screen Shot 2022-05-25 at 4 24 47 PM](https://user-images.githubusercontent.com/3821698/170785445-20671da4-8c5f-4529-afe7-4f3d9282bc13.png)

The error makes sense because `cleanUpWebView` removes `WKWebView` from its super view which probably affects the layout.

Reply:
If you look at the other methods, I wonder if it might be better (just for consistency sakes) to use `addUIBlock` instead of doing a dispatch_async

Reply:
It also queues the work on a different queue, and seems to be the natural fit here.